ui/src/authentication/components/LoginWithOauthButton/LoginWithOauthButton.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { Button } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { authenticationService } from '@authentication/services';
+
+export default class LoginWithOauthButton extends Component {
```

Comment:
Looks extremely similar with `LoginRedirectButton`, could we use one `LoginButton`?